app/views/unattended/provisioning_templates/snippet/preseed_netplan_setup.erb

```diff
@@ -23,7 +23,7 @@ oses:
 <%#
 ##### Processing bond-interfaces #####
 -%>
-<%- found = false -%>
+<%- found = false; id = 0 -%>
 <%- @host.bond_interfaces.each do | bond | -%>
   <%- bonding_interfaces.push(bond.identifier) -%>
   <%- if !found -%>
```

Comment:
If you have an `id` you can get rid of `found`
```suggestion
  <%- if id == 0 -%>
```

Reply:
thanks. make sense!